product-service/serverless.ts

```diff
@@ -26,6 +25,11 @@ const serverlessConfiguration: Serverless = {
     },
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      PG_HOST: 'shop.c85cvsfl0zex.eu-west-1.rds.amazonaws.com',
+      RG_PORT: 5432,
+      PG_DATABASE: 'shop',
+      PG_USERNAME: 'postgres',
+      PG_PASSWORD: '',
```

Comment:
It'll be better to import config params, especially admin credentials, from file that ignored by git, '.env' for example